main_routers/workshop_router.py

```
@@ -0,0 +1,1699 @@
+        return JSONResponse(content={
+            "success": False,
+            "error": str(e),
+            "message": "检查上传状态时发生错误"
+        }, status_code=500)
+
+
+@router.get('/read-file')
+async def read_workshop_file(path: str):
+    """读取创意工坊文件内容"""
+    try:
+        logger.info(f"读取创意工坊文件请求，路径: {path}")
```

Ruff (0.14.8)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)